config/routes.rb

```diff
@@ -297,6 +297,8 @@
   get "/feed" => "articles#feed", as: "feed", defaults: { format: "rss" }
   get "/feed/:username" => "articles#feed",
       as: "user_feed", defaults: { format: "rss" }
+  get "/feed/:tag" => "articles#feed",
```

Comment:
I'm confused—how does th routing differentiate between `:tag` and `:username` in terms of params here?

Reply:
@benhalpern yep good point, it definitely wouldn't and is just going to go by order. I had been using `feed?tag={tag}` but I'll move to `feed/tag/:tag` instead. It might make sense for username to do the same, but in the interest of not breaking existing feeds I'll leave as-is!